OpenEdX/Managers/PushNotificationsManager/Listeners/FCMListener.swift

```diff
@@ -6,11 +6,28 @@
 //
 
 import Foundation
+import Swinject
+import FirebaseMessaging
 
 class FCMListener: PushNotificationsListener {
+    
+    private lazy var deepLinkManager: DeepLinkManager? = {
+        Container.shared.resolve(DeepLinkManager.self)
+    }()
```

Comment:
How about passing this as a parameter, thoughts?

Reply:
Sure, why not
Done ✅